src/Core/Services/KeyConnectorService.cs

```diff
@@ -24,11 +24,11 @@ public class KeyConnectorService : IKeyConnectorService
             _organizationService = organizationService;
         }
 
-        public async Task GetAndSetKey(string url)
+        public async Task GetAndSetMasterKeyAsync(string url)
         {
             try
             {
-                var masterKeyResponse = await _apiService.GetMasterKeyFromKeyConnector(url);
+                var masterKeyResponse = await _apiService.GetMasterKeyFromKeyConnectorAsync(url);
                 var masterKeyArr = Convert.FromBase64String(masterKeyResponse.Key);
                 var masterKey = new MasterKey(masterKeyArr);
                 await _cryptoService.SetMasterKeyAsync(masterKey);
```

Comment:
⛏️ I see that in some places the suffix `...Bytes` it's being used and in others `...Arr`. We should be consistent on this and IMO `bytes` represents better what the variable type is (it's not an array of objects but a specific one of `byte`s).

⛏️ I'm not entirely convinced by the method name, given that the method doesn't return anything so `Get...` doesn't feel well to me. Maybe `UpdateMasterKeyFromUrlAsync(url)` or `UpdateMasterKeyFromAsync(url)` could be better, what do you think?

🤔 Sorry I can't get the catch section highlighted, but does the user need a special message (and localized) to be shown or is the generic one ok? Asking this because a generic `Exception` is being thrown.